server/sonar-web/src/main/js/apps/quality-profiles/components/ProfileContainer.tsx

```diff
@@ -17,32 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-import React from 'react';
+import * as React from 'react';
 import Helmet from 'react-helmet';
 import ProfileNotFound from './ProfileNotFound';
 import ProfileHeader from '../details/ProfileHeader';
-/*:: import type { Profile } from '../propTypes'; */
+import { IProfile } from '../types';
 
-/*::
-type Props = {
-  canAdmin: boolean,
-  children: React.Element<*>,
+interface Props {
+  canAdmin: boolean;
+  children: React.ReactElement<any>;
   location: {
-    pathname: string,
-    query: { key?: string, language: string, name: string }
-  },
-  onRequestFail: Object => void,
-  organization: ?string,
-  profiles: Array<Profile>,
-  router: { replace: ({}) => void },
-  updateProfiles: () => Promise<*>
-};
-*/
-
-export default class ProfileContainer extends React.PureComponent {
-  /*:: props: Props; */
+    pathname: string;
+    query: { key?: string; language: string; name: string };
+  };
+  onRequestFail: (reasong: any) => void;
+  organization: string | null;
+  profiles: IProfile[];
+  router: { replace: ({}) => void };
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this pattern to not be empty. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS3799)